framework/base/Security.php

```diff
@@ -31,6 +31,10 @@
 class Security extends Component
 {
     /**
+     * Buffer size for reading from /dev/random or /dev/urandom
+     */
+    const RANDOM_FILE_BUFFER = 8;
```

Comment:
If you docblock like this then it becomes part of the published API for programmers. I don't think that's wise. I would put it at line ~430 and put the comment in the method where you use the const.